flake-registry.json

```diff
@@ -221,28 +11,6 @@
         "type": "github"
       }
     },
-    {
-      "from": {
-        "id": "nix-darwin",
-        "type": "indirect"
-      },
-      "to": {
-        "owner": "LnL7",
-        "repo": "nix-darwin",
-        "type": "github"
-      }
-    },
-    {
-      "from": {
-        "id": "nixops",
-        "type": "indirect"
-      },
-      "to": {
-        "owner": "NixOS",
-        "repo": "nixops",
-        "type": "github"
-      }
-    },
```

Comment:
nix-darwin and nixops are both important projects

Reply:
For nix-darwin I can agree, similar to how I agree with HM, though nixops, I am not sure about, given the state it seems to be in.

I am never sure if it is alive or not.

Reply:
It's still an "official" project, but I agree that it's debatable. At the end, that kind of decision is probably subjective.

Reply:
It is always subjective, thats why we really need some clear guidelines and policy as asked for in #25.

Reply:
Both are true. Clear guidelines are useful because they help create a shared understanding of what should be included or not. But there are always grey where judgement calls are necessary.